reana-ui/src/pages/Login.js

```diff
@@ -33,8 +33,32 @@ import {
 } from "semantic-ui-react";
 import LogoImg from "../images/logo-reana.svg";
 
+const URL = "http://reana-dev.cern.ch/api/users?";
+
 export default class LoginPage extends Component {
+
+  state = {
+    email: '',
+    token: '',
+  };
+
+  handleChange = (e, { name, value }) => {
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = () => {
+    const { email, token } = this.state;
+
+    fetch(URL + "access_token=" + token)
```

Comment:
I think here starts the complication which I make reference in #19. To correctly handle errors we should resolve the promise and check if the status is the one we expect. `fetch` itself it is going to throw and exception only when there are network problems, but not when you for instance, get a 404. See the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful) about correctly checking if `fetch` works. @reanahub/developers